apps/web/client/src/app/project/[id]/_components/top-bar/github-export/dropdown/index.tsx

```
@@ -0,0 +1,250 @@
+                    name: existingConnection.repositoryName,
+                    full_name: existingConnection.fullName,
+                    html_url: existingConnection.repositoryUrl,
+                    description: '',
+                    private: true,
+                    default_branch: existingConnection.branch || 'main',
+                    clone_url: '',
+                    owner: {
+                        login: existingConnection.repositoryOwner,
+                        avatar_url: '',
+                    },
+                } : null);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)